explorer/package.json

```
@@ -39,8 +39,9 @@
     "@heroicons/react": "^2.1.1",
     "@netlify/edge-functions": "^2.11.1",
     "@next/third-parties": "^14.2.3",
-    "@nivo/core": "^0.84.0",
-    "@nivo/line": "^0.84.0",
+    "@nivo/bar": "^0.88.0",
+    "@nivo/core": "^0.88.0",
+    "@nivo/line": "^0.88.0",
     "@nivo/pie": "^0.84.0",
```

```suggestion
    "@nivo/pie": "^0.88.0",
```
Let's take the opportunity to update all the nivo dependencies 👍

---

updated the version